src/lib/features/project-insights/project-insights-service.ts

```diff
@@ -0,0 +1,164 @@
+import type {
+    IFeatureToggleStore,
+    IFeatureTypeStore,
+    IProjectHealth,
+    IProjectStore,
+    IUnleashStores,
+} from '../../types';
+import type FeatureToggleService from '../feature-toggle/feature-toggle-service';
+import { calculateAverageTimeToProd } from '../feature-toggle/time-to-production/time-to-production';
+import type { IProjectStatsStore } from '../../types/stores/project-stats-store-type';
+import type { ProjectDoraMetricsSchema } from '../../openapi';
+import { calculateProjectHealth } from '../../domain/project-health/project-health';
+
+export class ProjectInsightsService {
```

Comment:
moved code from project-service

Reply:
this is view model/external read model but since we still use shared services/index.ts I stick to the services name